CHANGELOG.md

```diff
@@ -8,27 +8,37 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 ### Added
-- Added an analytical test for RPCholesky, and associated documentation in `tests.
-  unit.test_solvers`.(https://github.com/gchq/coreax/pull/822)
-- Added a unit test for RPCholesky to check whether the coreset has duplicates.(https://github.com/gchq/coreax/pull/836)
+
+- Added an analytical test for RPCholesky, and associated documentation in
+  `tests. unit.test_solvers`. (https://github.com/gchq/coreax/pull/822)
+- Added a unit test for RPCholesky to check whether the coreset has duplicates.
+  (https://github.com/gchq/coreax/pull/836)
 
 ### Fixed
+
 - `RPCholesky.reduce` in `coreax.solvers.coresubset` now computes the iteration step
-  correctly.(https://github.com/gchq/coreax/pull/825)
+  correctly. (https://github.com/gchq/coreax/pull/825)
 - `RPCholesky.reduce` in `coreax.solvers.coresubset` now does not produce duplicate
-  points in the coreset.(https://github.com/gchq/coreax/pull/836)
+  points in the coreset. (https://github.com/gchq/coreax/pull/836)
+- Build includes sub-packages. (https://github.com/gchq/coreax/pull/845)
 
 ### Changed
+
 -
 
 ### Removed
+
 -
 
 ### Deprecated
+
 -
 
+
 ## [0.3.0]
```

Comment:
[Keep a Changelog](https://keepachangelog.com/en/1.0.0/#yanked) recommends having a big obvious `[YANKED]` tag in the title

Reply:
Good point @rg936672. I'll not merge, till this is added